cli/utils/calculate-rewards.ts

```diff
@@ -91,19 +95,26 @@ export function processAttestationCompletion(
     issuers[account].push(issuer)
     state.attestationCompletions[account] += 1
   }
+}
 
-  if (state.startedBlockBalanceTracking && state.attestationCompletions[account] === 3) {
-    // Just completed
-    state.balancesByBlock[account] = {
-      balances: [state.balances[account] || new BigNumber(0)],
-      blocks: [event.blockNumber],
-    }
+export function processAccountWalletAddressSet(walletAssociations: WalletAssociations, event: EventLog) {
+  const account = event.returnValues.account
+  const wallet = event.returnValues.walletAddress
+  if (!walletAssociations[wallet]) walletAssociations[wallet] = [wallet] // the wallet itself may be verified
```

Comment:
I haven't seen this style used in our codebase and so just flagging the question of whether we want to include this in our linter or not. cc @nambrot

Reply:
I like to make my code as boxy as possible 😂 can nest this though, if the org prefers that style

Reply:
I don't have a strong preference, just wanted to check